selfdrive/car/subaru/values.py

```diff
@@ -615,3 +615,4 @@ def init_make(self, CP: car.CarParams):
 LKAS_ANGLE = {CAR.FORESTER_2022, CAR.OUTBACK_2023}
 GLOBAL_GEN2 = {CAR.OUTBACK, CAR.LEGACY, CAR.OUTBACK_2023}
 PREGLOBAL_CARS = {CAR.FORESTER_PREGLOBAL, CAR.LEGACY_PREGLOBAL, CAR.OUTBACK_PREGLOBAL, CAR.OUTBACK_PREGLOBAL_2018}
+STEER_LIMITED_2020 = GLOBAL_GEN2 | {CAR.IMPREZA_2020}
```

Comment:
why is 2020 in the name?

Reply:
its 2020+ cars that have this limit

Reply:
the 90 degree limit or high angle rate, or both? a comment might be better

Reply:
90 deg fault it pretty rare in normal driving but I can check the data. I'll just name it STEER_RATE_LIMITED for now and we can come back to the 90 deg fault later